source/_includes/pup40_platforms_redhat_like.markdown

```diff
@@ -1,10 +1,10 @@
 We publish official packages and run automated testing on the following versions for Puppet 4.x:
 
-* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
 * Enterprise Linux 7 (also supported by [Puppet Enterprise][peinstall])
-
+* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
+* Enterprise Linux 5 (also supported by [Puppet Enterprise][peinstall])
```

Comment:
ah, yeah, definitely :(

https://confluence.puppetlabs.com/display/PROD/FOSS+Supported+Platforms+for+PC1

cc/ @underscorgan